contrib/endpoints/src/api_manager/config.h

```diff
@@ -40,9 +40,19 @@ class Config {
   // any of protobuf format json, binary or text. It can be nullptr if there is
   // not server_config.
   static std::unique_ptr<Config> Create(ApiManagerEnvInterface *env,
+                                        const std::string &service_config);
+  // For unit test only
```

Comment:
I am wary of code that is marked as "for test only". This usually means the test has a different code path than what is used in production.